app/service/data_svc.py

```diff
@@ -204,10 +204,12 @@ def __init__(self):
                             self.log.debug('Ability no longer exists on disk, removing: %s' % existing.unique)
                             await self.remove('abilities', match=dict(unique=existing.unique))
                         if existing.payload:
-                            _, path = await self.get_service('file_svc').find_file_path(existing.payload)
-                            if not path:
-                                self.log.error('Payload referenced in %s but not found: %s' %
-                                               (existing.ability_id, existing.payload))
+                            payloads = existing.payload.split(',')
+                            for payload in payloads:
```

Comment:
perfect-- I thought this had gotten dropped at some point - thanks for adding this back in!